lms/djangoapps/courseware/views/views.py

```diff
@@ -2127,11 +2144,15 @@ def get_financial_aid_courses(user):
                     Q(_expiration_datetime__isnull=True) | Q(_expiration_datetime__gt=datetime.now(UTC)),
                     course_id=enrollment.course_id,
                     mode_slug=CourseMode.VERIFIED).exists():
-
+            default = False
+            # This is a workaround to set course_id before disabling the field in case of new financial assistance flow.
+            if str(enrollment.course_overview) == course_id:
+                default = True
             financial_aid_courses.append(
                 {
                     'name': enrollment.course_overview.display_name,
-                    'value': str(enrollment.course_id)
+                    'value': str(enrollment.course_id),
+                    'default': default
                 }
```

Comment:
Shouldn't we just return the single course if course_id is provided, rather than loop over all enrollments?

Reply:
We can do that. I'll limit this for one course_id then.